__tests__/app.test.js

```diff
@@ -81,7 +79,30 @@ describe('GET /api/articles/:article_id', () => {
 
 });
 
-describe('GET /api/topics ERR HANDLING', () => {
+describe('GET /api/articles', () => {
+    it('200: responds with an articles array of article objects, sorted by date, each of which should have the expected properties', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toHaveLength(13)
+            body.forEach((article) => {
+                expect(Object.keys(article)).toContain('author') 
+                expect(Object.keys(article)).toContain('title') 
+                expect(Object.keys(article)).toContain('article_id') 
+                expect(Object.keys(article)).toContain('topic') 
+                expect(Object.keys(article)).toContain('created_at') 
+                expect(Object.keys(article)).toContain('article_img_url') 
+                expect(Object.keys(article)).toContain('votes') 
+                expect(Object.keys(article)).toContain('comment_count') 
```

Comment:
You may also want to test the comment count is correct on one of the objects returned